influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -1082,108 +1081,69 @@ mod tests {
         )
         .await;
 
+        let db_name = "foo";
         // do three writes to force a snapshot
-        let _ = write_buffer
-            .write_lp(
-                NamespaceName::new("foo").unwrap(),
-                "cpu bar=1",
-                Time::from_timestamp(10, 0).unwrap(),
-                false,
-                Precision::Nanosecond,
-            )
-            .await
-            .unwrap();
-        let _ = write_buffer
-            .write_lp(
-                NamespaceName::new("foo").unwrap(),
-                "cpu bar=2",
-                Time::from_timestamp(20, 0).unwrap(),
-                false,
-                Precision::Nanosecond,
-            )
-            .await
-            .unwrap();
-        let _ = write_buffer
-            .write_lp(
-                NamespaceName::new("foo").unwrap(),
-                "cpu bar=3",
-                Time::from_timestamp(30, 0).unwrap(),
-                false,
-                Precision::Nanosecond,
-            )
-            .await
-            .unwrap();
+        do_writes(
```

Comment:
I'm glad we have this method now. Thanks for updating these tests to use the new way of doing a write